### PR TITLE
SAK-44168 Kernel antisamy should not sanitize content item links in the RTE

### PR DESCRIPTION
**Pull request by ern**

### CODE REVIEW

**Comment by ern** [comment]
This only happens using the HIGH antisamy policy as the LOW policy allows anything.

File: kernel/kernel-impl/src/test/java/org/sakaiproject/util/impl/FormattedTextTest.java [hunk 1]
```diff
@@ -1224,5 +1226,13 @@ public void getHtmlBodyTest() {
         Assert.assertEquals("", result);
     }
 
+    @Test
+    public void testLocalIframeSrc() {
+        String contentItemIframe = "<iframe allowfullscreen=\"true\" class=\"lti-iframe\" height=\"402\" mozallowfullscreen=\"true\" src=\"http://localhost:8080/access/basiclti/site/0f68e843-1f0c-473d-b469-852a49ea0f05/content:62\" title=\"Test LTI Content Item Iframe\" webkitallowfullscreen=\"true\" width=\"608\"></iframe>";
```
Comment (adrianfish) [inline kernel/kernel-impl/src/test/java/org/sakaiproject/util/impl/FormattedTextTest.java:1231]: I'd extract that localhost url into a variable, poysonally.

**Review by adrianfish — APPROVED** [review]
GTG.

**Comment by axxter99** [comment]
Trunk (/localhost:8080, trunk-mysql.nightly.sakaiproject.org, etc): 
2020-08-29 06:54:48,486  WARN main org.sakaiproject.util.impl.FormattedTextImpl - XML failure while updating policy file: /home/dhorwitz/Sakai/file:/home/dhorwitz/Sakai/apache-tomcat-9.0.37/components/sakai-kernel-impl/WEB-INF/lib/sakai-kernel-impl-21-SNAPSHOT.jar!/antisamy/high-security-policy.xml
java.io.FileNotFoundException: /home/dhorwitz/Sakai/file:/home/dhorwitz/Sakai/apache-tomcat-9.0.37/components/sakai-kernel-impl/WEB-INF/lib/sakai-kernel-impl-21-SNAPSHOT.jar!/antisamy/high-security-policy.xml (No such file or directory)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)

**Pull request merged**